vars/folioEurekaApp.groovy

```diff
@@ -19,12 +21,11 @@ def generateAppPlatformCompleteDescriptor() {
 }
```

Comment:
Enhancement: in my opinion it would be better to combine 'generateAppPlatformMinimalDescriptor' and 'generateAppPlatformCompleteDescriptor' functions into the one function and pass Application Name via function parameter.